### DISCUSSION TITLE
MongoDB is non-free software, consider changing database

### DISCUSSION THREAD

**Discussion by gnusupport** [payloadcms/payload#856] [Feature Requests & Ideas]
As MongoDB is non-free software it is not included in many GNU/Linux distributions.

PayLoad requires MongoDB  which is non-free software and thus cannot be included in GNU/Linux distributions.

Please consider changing the non-free database software to free software database.

**Comment by jmikrut** [comment]
Hey @gnusupport — we do have a roadmap item open for supporting more databases:

https://github.com/payloadcms/payload/discussions/287
